packages/babel-plugin-transform-regexp-modifiers/test/fixtures/syntax/with-u-flag/output.js

```diff
@@ -1,2 +1,2 @@
 /(?:(?:^|(?<=[\n\r\u2028\u2029]))[A-Za-z\u017F\u212A])/;
-/(?:^[a-z](?:[\0-\uD7FF\uE000-\uFFFF]|[\uD800-\uDBFF][\uDC00-\uDFFF]|[\uD800-\uDBFF](?![\uDC00-\uDFFF])|(?:[^\uD800-\uDBFF]|^)[\uDC00-\uDFFF]))((?:^|(?<=[\n\r\u2028\u2029]))[A-Za-z\u017F\u212A](?:[\0-\uD7FF\uE000-\uFFFF]|[\uD800-\uDBFF][\uDC00-\uDFFF]|[\uD800-\uDBFF](?![\uDC00-\uDFFF])|(?:[^\uD800-\uDBFF]|^)[\uDC00-\uDFFF]))/;
+/(?:^[a-z](?:[\0-\t\x0B\f\x0E-\u2027\u202A-\uD7FF\uE000-\uFFFF]|[\uD800-\uDBFF][\uDC00-\uDFFF]|[\uD800-\uDBFF](?![\uDC00-\uDFFF])|(?:[^\uD800-\uDBFF]|^)[\uDC00-\uDFFF]))((?:^|(?<=[\n\r\u2028\u2029]))[A-Za-z\u017F\u212A](?:[\0-\uD7FF\uE000-\uFFFF]|[\uD800-\uDBFF][\uDC00-\uDFFF]|[\uD800-\uDBFF](?![\uDC00-\uDFFF])|(?:[^\uD800-\uDBFF]|^)[\uDC00-\uDFFF]))/;
```

Comment:
This is a bug fix. Previously regexpu-core does not correctly handle the `.` within the `(?-s:)` group, the dot was treated as if it were still in the dotAll mode (https://github.com/mathiasbynens/regexpu-core/issues/94).